token_auth/views/login.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render, redirect
+from rest_framework import generics, status, views, permissions
+from token_auth.serializers.login import LoginSerializer
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
+from django.contrib.sites.shortcuts import get_current_site
+
+"""
+Function for user login after registration
+"""
+
+class LoginView(generics.GenericAPIView):
+    serializer_class = LoginSerializer
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
I don't think that you need generic view here
```suggestion
class LoginView(APIView):
    def post(self, request):
        serializer = LoginSerializer(data=request.data)
        serializer.is_valid(raise_exception=True)
        return Response(serializer.data, status=status.HTTP_200_OK)

```